inc/uploadcare_settings.php

```diff
@@ -76,6 +78,7 @@
     $uploadcare_finetuning = get_option('uploadcare_finetuning');
     $uploadcare_source_tabs = get_option('uploadcare_source_tabs', $tab_defaults);
     $uploadcare_tab_effects = get_option('uploadcare_tab_effects', $effects_defaults);
+    $uploadcare_lifetime = get_option('uploadcare_lifetime', '0');
```

Comment:
```suggestion
    $uploadcare_lifetime = get_option('uploadcare_upload_lifetime', '0');
```

we'll probably want to add support for Authenticater URLs as well, with something like `uploadcare_cdn_lifetime` option name.

Or `uploadcare_upload_sig_lifetime` and `uploadcare_cdn_sig_lifetime` to make it more verbose.